setup.py

```diff
@@ -89,7 +89,7 @@ def read_reqs(req_file):
     packages=find_packages(),
     description=("Elastic Search Connectors."),
     long_description=long_description,
-    author="Ingestion Team",
+    author="Search Extract and Transform Team",
     author_email="tarek@ziade.org",
```

Comment:
I wonder if the `author_email` should be updated at any point.

Reply:
Yeah, I had the same question on my mind, wanted to check with PM.